cmd/spec/example_source.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+)
+
+var SourceListResponse = []payloads.SourceResponse{{
+	ID:   "654321",
+	Name: "My AWS account",
+}, {
+	ID:   "543621",
+	Name: "My other AWS account",
+}}
+
+var SourceUploadInfoAWSResponse = payloads.SourceUploadInfoResponse{
+	Provider: "aws",
+	AwsInfo:  &clients.AccountDetailsAWS{AccountID: "78462784632"},
+}
+
+var SourceUploadInfoAzureResponse = payloads.SourceUploadInfoResponse{
```

Comment:
I'm confused :thinking:
This generates example where we have key `azureinfo`, but I don't see where the type is defined, but schema defines a key `azure`, so there is discrepancy in the key names, but I don't see where it comes from. Are we using the same types for The example response and the schema generating?

Reply:
![image](https://github.com/RHEnVision/provisioning-backend/assets/2884324/b25414ee-3089-40cd-ad6c-6ab0fe829c1b)

Reply:
Yeah this is a typical error I have seen many times now:

```diff
diff --git a/internal/payloads/sources_payload.go b/internal/payloads/sources_payload.go
index cb3dd40..2da271e 100644
--- a/internal/payloads/sources_payload.go
+++ b/internal/payloads/sources_payload.go
@@ -37,9 +37,9 @@ func NewListSourcesResponse(sourceList []*clients.Source) []render.Renderer {
 }
 
 type SourceUploadInfoResponse struct {
-       Provider  string                       `json:"provider"`
-       AwsInfo   *clients.AccountDetailsAWS   `json:"aws" nullable:"true"`
-       AzureInfo *clients.AccountDetailsAzure `json:"azure" nullable:"true"`
+       Provider  string                       `json:"provider" yaml:"provider"`
+       AwsInfo   *clients.AccountDetailsAWS   `json:"aws" nullable:"true" yaml:"aws"`
+       AzureInfo *clients.AccountDetailsAzure `json:"azure" nullable:"true" yaml:"azure"`
 }
```

This should fix it, pushing.

Reply:
That helped, thanks! :rocket: